### DISCUSSION TITLE
Is it possible to provide dashboard grafana for overview, logs and trace as we can see on screenshot

### DISCUSSION THREAD

**Discussion by BenjaminBusvel** [grafana/grafana-opentelemetry-java#527] [Q&A]
Hello,

Is it possible to shared your dashboard grafana for overview, logs and trace as we can see on screenshot ?

Best regards,

**Comment by zeitlinger** [comment]
it's from https://grafana.com/docs/grafana-cloud/monitor-applications/application-observability/ - but some parts are here: https://github.com/grafana/docker-otel-lgtm/tree/main/docker

@gouthamve has some more...

**Reply by BenjaminBusvel** [reply]
Thank you for your feedback and your links. In fact I was looking for your trace and log dashboards which are not in the indicated repositories. If you cannot provide these dashboards, would you have the logQL and other queries that you made on these dashboards? thanks in advance

**Reply by zeitlinger** [reply]
it's context dependent - it's probably easiest to try it out in application observability to see how it works. here's an example:

logs:

```
{exporter="OTLP", job="otel-demo/adservice"} | json | resources_deployment_environment=~"production" | line_format `[1m{{if .level }}{{alignRight 5 .level}}{{else if .severity}}{{alignRight 5 .severity}}{{end}}[0m [90m[{{alignRight 10 .resources_service_instance_id}}{{if .attributes_thread_name}}/{{alignRight 20 .attributes_thread_name}}{{else if eq "java" .resources_telemetry_sdk_language }}                    {{end}}][0m [36m{{if .instrumentation_scope_name }}{{alignRight 40 .instrumentation_scope_name}}{{end}}[0m {{.body}} {{if .traceid}} [37m[3m[traceid={{.traceid}}]{{end}}`
```

traces:

```
{resource.service.name="adservice" && kind=server && resource.deployment.environment=~"production" && resource.service.namespace="otel-demo"}
```
